packs/bit-bucket/actions/create_ssh_key.yaml

```diff
@@ -0,0 +1,18 @@
+name: create_ssh_key
+runner_type: run-python
+description: Create SSH key
```

Comment:
I needed to check the Python library documentation to figure out what this action does.

Imo, we should change the description so it says something like "Associate an ssh key with your account and return it." since this method / action associated an existing public key with your account.